tests/product/base_product_case.py

```diff
@@ -85,10 +85,7 @@ class BaseProductTestCase(BaseTestCase):
 query.max-memory-per-node=512MB
 query.max-memory=50GB\n"""
 
-    # The two strings below (down_node_connection_string and status_down_node_string) aggregate
-    # all possible error messages one might encounter when trying to perform an action when a
-    # node is not accessible. The variety in error messages comes from differences in the OS.
-    down_node_connection_string = r'(\nWarning: (\[%(host)s\] )?Low level ' \
+    network_unreachable = r'(\nWarning: (\[%(host)s\] )?Low level ' \
```

Comment:
Is this one is still valid once you ported tests to use network? I thought only one of these two errors should be expected.